packages/core/src/node-utils/framework.ts

```diff
@@ -1,5 +1,5 @@
 import { retrieveJsonPackage, JsonPackage } from "./pkg";
-import { Frameworks, SupportedFrameworks } from "../models/Framework";
+import { Frameworks, SupportedFrameworks, Version } from "../models/Framework";
```

Comment:
do we need this ? 

later on it's 
```
version: Version; 
```
which could be `version?: string`

```
```

Reply:
That's what I had initially, but it was repeated quite often. I thought this made it clearer. Can change if you prefer

Reply:
yes, 👍